controller/release.go

```diff
@@ -63,6 +63,13 @@ func (r *ReleaseRepo) Add(data interface{}) error {
 		release.ArtifactIDs = []string{release.LegacyArtifactID}
 	}
 
+	if _, ok := release.Env[""]; ok {
+		return ct.ValidationError{
+			Field:   "env",
+			Message: "you can't create an env var with an empty key",
```

Comment:
Given this is usually done by mistake, it would be useful to include the value in the response, for example:

```
$ flynn env set =BAR
validation_error: env you can't create an env var with an empty key (tried to set ""="BAR")
```